examples/gravitational_waves_detection.ipynb

```diff
@@ -0,0 +1,438 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Gravitational wave detection\n",
+    "Chrisopher Bresten and Jae-Hun Jung propose to include topological features in a CNN classifier for gravitational waves detection. Adapted from their article, this notebook showcases an application of ideas from the [Topology of time series example](https://giotto-ai.github.io/gtda-docs/latest/notebooks/topology_time_series.html).\n",
+    "\n",
+    "If you are looking at a static version of this notebook and would like to run its contents, head over to [GitHub](https://github.com/giotto-ai/giotto-tda/blob/master/examples/gravitational_waves_detection.ipynb) and download the source.\n",
+    "\n",
+    "## Useful references\n",
+    "\n",
+    "* [Topology of time series example](https://giotto-ai.github.io/gtda-docs/latest/notebooks/topology_time_series.html), in which the *Takens embedding* technique is explained on simple examples and in detail.\n",
```

Comment:
Suggestion: remove "example" from text in square brackets. Also another suggestion: "in which the *Takens embedding* technique used here is explained in detail and illustrated via simple examples."